app/scripts/notification.js

```diff
@@ -0,0 +1,72 @@
+const NotificationRoot = require('../../ui/app/pages/notification-root')
+const startPopup = require('./popup-core')
+const PortStream = require('extension-port-stream')
+const { getEnvironmentType } = require('./lib/util')
+const { ENVIRONMENT_TYPE_NOTIFICATION, ENVIRONMENT_TYPE_FULLSCREEN } = require('./lib/enums')
+const extension = require('extensionizer')
+const ExtensionPlatform = require('./platforms/extension')
+const NotificationManager = require('./lib/notification-manager')
+const notificationManager = new NotificationManager()
+const setupSentry = require('./lib/setupSentry')
+const log = require('loglevel')
+
+start().catch(log.error)
+
+async function start () {
```

Comment:
This function seems fairly similar to the one in `ui.js`. We should refactor them into a shared function - which maybe can also go in `popup-core`?